### ISSUE TITLE
[Doc] [Improvement] Provide tutorials on using GraphAr libraries in documentation

### ISSUE THREAD

**Issue by lixueclaire** [alibaba/GraphAr#168]
**Is your feature request related to a problem? Please describe.**
The GraphAr project currently offers the C++ library and the Spark library, with "Getting Started" and  "API References" available in the [documentation](https://alibaba.github.io/GraphAr/index.html). However, these references may not suffice for developers to quickly and intuitively understand the usage of the libraries.

**Describe the solution you'd like**
To address this, we propose adding tutorials on how to use the libraries in the documentation. These tutorials can be organized by programming language and separated into independent units for introducing different classes. As a first step, we can add tutorials on Info classes, such as GraphInfo, VertexInfo, and EdgeInfo.
We recommend referring to [Apache Arrow's documentation](https://arrow.apache.org/docs/cpp/user_guide.html) for writing the tutorials.

**Comment by acezen** [comment]
some tutorial need to add:
[ ] How to convert csv format dataset to GraphAr with C++ SDK
[ ] How to convert csv format dataset to GraphAr with Spark SDK

**Comment by silspo** [comment]
Hi , Would love to work on this , as it would also help me understand the stack faster while I build the tutorial. Can I pick this one up ? Thank You!

**Comment by Thespica** [comment]
> Hi , Would love to work on this , as it would also help me understand the stack faster while I build the tutorial. Can I pick this one up ? Thank You!

Of course! Thanks for your interest in GraphAr, feel free to reach out to us if you have any problem!
